README.md

```diff
@@ -1 +1,88 @@
-# node-modules-object-lab
\ No newline at end of file
+# Using Object Module
+
+## Objectives
+
+1. Write a simple non-npm module with module.exports = {}
+1. Import your module from another file
+1. Invoke a method from the module
+
+
+## Introduction
+
+We have some customer data for testing our application before we implement the real database or REST API. It's a bad idea to store this data in the code, so you're task is to organize the data as a separate file. This way it will be easier to switch from this customer data to some other source later.
```

Comment:
Nit (as in, low priority and something we can fix quickly/later): "you're" -> "your"